src/libraries/System.Security.Cryptography.Xml/tests/CanonicalXmlEntityReferenceTests.cs

```diff
@@ -0,0 +1,49 @@
+using System.Security.Cryptography;
+using System.Text;
+using System.Xml;
+using Xunit;
+
+namespace System.Security.Cryptography.Xml.Tests
+{
+    public class CanonicalXmlEntityReferenceTests
```

Comment:
I don't see anything in this class that uses instance members besides methods. Can we mark this class as `static` (as well as all of the test methods)?

Same feedback to the other new file.